app/controllers/reservations_controller.rb

```diff
@@ -330,7 +330,7 @@ def checkout # rubocop:disable all
     # prep for receipt page and exit
     @check_in_set = []
     @check_out_set = checked_out_reservations
-    render('receipt') && return
+    render('receipt', layout: 'application_with_search_sidebar') && return
```

Comment:
So, if we look at the [last version](https://github.com/YaleSTC/reservations/blob/v3.4.10/app/controllers/reservations_controller.rb#L5), it looks like we were using the sidebar layout for *all* ReservationsController views, not just the receipt pages. I think that going with that here would be fine too; Bootstrap should adjust the Reservation show views to make everything fit nicely. I'll try this locally to make sure it looks ok and I'll update this thread.